source/Calamari.Tests/KubernetesFixtures/InstallTools.cs

```diff
@@ -58,12 +59,9 @@ public async Task Install()
                 KubectlExecutable = await DownloadCli("Kubectl",
                     async () =>
                     {
```

Comment:
Switched back to the latest version of kubectl, this is in the context of tests and should help us spot issues earlier. This failed previously on an kubectl and aws-cli version mismatch. https://github.com/aws/aws-cli/issues/6920